roles/visualmesh_fake.role

```diff
@@ -0,0 +1,23 @@
+nuclear_role(
+  # FileWatcher, ConsoleLogHandler and Signal Catcher Must Go First. KinematicsConfiguration usually goes after these
+  # and without it many roles do not run
+  support::SignalCatcher
+  support::logging::ConsoleLogHandler
+  extension::FileWatcher
+  # This must come first as it emits config which many roles depend on (e.g. SensorFilter, WalkEngine)
+  motion::KinematicsConfiguration
+  # Support
+  network::NUClearNet
+  support::NUsight
+  # support::VirtualCamera
+  support::configuration::SoccerConfig
+  # Sensors
+  input::FakeCamera
+  input::ImageDecompressor
+  # Vision
+  vision::VisualMesh
+  vision::GreenHorizonDetector
+  vision::BallDetector
+  vision::GoalDetector
+  # Data logger support::logging::DataLogging
```

Comment:
Why is this commented? I don't think it's meant to be

Reply:
Was commented in `visualmesh.role` when I duplicated it to make the fake role :P

Reply:
Hmmmmmm